src/app/src/main/kotlin/org/vaccineimpact/api/app/logic/BurdenEstimateLogic.kt

```diff
@@ -172,26 +169,40 @@ class RepositoriesBurdenEstimateLogic(private val modellingGroupRepository: Mode
         {
             throw UnknownObjectError(touchstoneVersion.id, TouchstoneVersion::class)
         }
-        //Check that scenario exists
+
         scenarioRepository.checkScenarioDescriptionExists(path.scenarioId)
     }
 
     private fun missingRows(countryMapEntry: Map.Entry<String, HashMap<Short, HashMap<Short, Boolean>>>): Boolean
     {
-        return countryMapEntry.value.any { a ->
-            a.value.any { y -> !y.value }
+        return countryMapEntry.value.any { age ->
+            age.value.any { year -> !year.value }
         }
     }
 
-    private fun rowErrorMessage(missingRows: Map<String, HashMap<Short, HashMap<Short, Boolean>>>): String
+    private fun rowErrorMessage(countriesWithMissingRows: Map<String, HashMap<Short, HashMap<Short, Boolean>>>): String
+    {
+        val countries = countriesWithMissingRows.keys
+        val firstCountryWithMissingData = countries.first()
+        val firstMissingAgesRow = countriesWithMissingRows.getValue(firstCountryWithMissingData)
+        val firstAgeWithMissingYears = firstMissingAgesRow.filter(::hasFalseEntry).keys.first()
+        val firstMissingYearsRow = firstMissingAgesRow.getValue(firstAgeWithMissingYears)
+        val firstMissingYear = firstMissingYearsRow.filter(::isFalse).keys.first()
+
+        val basicMessage = "Missing rows for ${countries.joinToString(", ")}"
+        val exampleRowMessage = "For example:\n$firstCountryWithMissingData, age $firstAgeWithMissingYears," +
+                " year $firstMissingYear"
+        return "$basicMessage\n$exampleRowMessage"
+    }
+
+    private fun isFalse(mapEntry: Map.Entry<Short, Boolean>): Boolean
+    {
+        return !mapEntry.value
+    }
+
+    private fun hasFalseEntry(mapEntry: Map.Entry<Short, HashMap<Short, Boolean>>): Boolean
     {
-        val countries = missingRows.keys
-        val message = "Missing rows for ${countries.joinToString(", ")}"
```

Comment:
This method was the problem. The `missingRows` argument is not actually a map containing only missing rows, rather it is a collection of all rows for countries that have at least one missing row. We can't assume that the first country/year/age combo is actually missing, we need to check for false values.

I've changed the method to do just that, changing the name of the `missingRows` argument as well to clarify. The other way to fix this would be to actually filter the whole nested hashmap to be only those rows which are missing before we pass it in to this method. But I think that would require more enumeration and result in more new objects being created (so be less performant). I haven't actually tested that though.